test/integration/steps/test_parent_steps.py

```diff
@@ -178,11 +143,14 @@ def a_standard_opening_time_change_event_is_valid():
     return context
 
 
-@given("a Changed Event is aligned with Dos", target_fixture="context")
-def a_change_event_is_valid_and_matches_dos():
-    context = {}
-    context["change_event"] = build_same_as_dos_change_event("pharmacy")
-    return context
+@given(parsers.parse('a "{org_type}" Changed Event is aligned with Dos'), target_fixture="context")
+def dos_event_from_scratch(org_type: str):
+    if org_type.lower() in ["pharmacy", "dentist"]:
```

Comment:
Can you standardise the org_type alias that you use all over the place. In some places it is Pharmacy, others PHARMACY or pharmacy. Everyplace being uppercase using a constant would resolve this. This is within the python code. I don't care if you have different org type style in the feature files.